.github/CODEOWNERS

```diff
@@ -237,6 +237,7 @@
 /core/templates/services/exploration-html-formatter.service*.ts @kevintab95
 /core/templates/services/image-upload-helper.service*.ts @kevintab95
 /core/templates/services/validators.service*.ts @kevintab95
+/core/templates/domain/entity_editor_browser_tabs_info/ @kevintab95
```

Comment:
Add this in alphabetically sorted order.

Reply:
done